src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -58,6 +68,7 @@ public void beforeEach(TestInfo info) {
     @AfterEach
     public void afterEach(TestInfo info) {
         TestUtils.printTrace(TestUtils.TraceSevLevel.INFO, this.getClass().getSimpleName() + "." + info.getDisplayName() + ". Exit");
+        cleanAndResetTerminal();
```

Comment:
You should perform all operations before you print "Exit."

Reply:
Changed